pkg/virt-operator/webhooks/kubevirt-update-admitter_test.go

```diff
@@ -155,6 +161,123 @@ var _ = Describe("Validating KubeVirtUpdate Admitter", func() {
 			Entry("1.123", "1.123"),
 		)
 	})
+
+	Context("deprecations", func() {
+		var admitter *KubeVirtUpdateAdmitter
+
+		BeforeEach(func() {
+			clusterConfig, _, _ := testutils.NewFakeClusterConfigUsingKVConfig(&v1.KubeVirtConfiguration{})
+			admitter = NewKubeVirtUpdateAdmitter(nil, clusterConfig)
+		})
+
+		admit := func(kubevirt v1.KubeVirt) *admissionv1.AdmissionResponse {
+			kvBytes, err := json.Marshal(kubevirt)
+			Expect(err).ToNot(HaveOccurred())
+
+			request := &admissionv1.AdmissionReview{
+				Request: &admissionv1.AdmissionRequest{
+					Resource: KubeVirtGroupVersionResource,
+					Object: runtime.RawExtension{
+						Raw: kvBytes,
+					},
+					OldObject: runtime.RawExtension{
+						Raw: kvBytes,
+					},
+					Operation: admissionv1.Update,
+				},
+			}
+			return admitter.Admit(request)
+		}
+
+		warn := true
+		warnNotExpected := false
```

Comment:
Make these at least const or better inline them?

Reply:
I made them const. What do you mean by inline? Do you mean not using variables just using the `true|false`? I find that hard to read sometimes and this should not hurt anybody.

Reply:
Yes, I meant just using `true|false` but making them const is fine too!